### ISSUE TITLE
Search not displaying

### ISSUE THREAD

**Issue by rogeralloymagnetic** [redhat-developer/developers.redhat.com#3126]
/search/?s=most-recent&f=type~videold/

**Comment by jordanpagewhite** [comment]
This should be covered by this ticket: https://github.com/redhat-developer/rhd-frontend/issues/184

Jason Porter and Luke have been working on a search solution.

**Comment by jordanpagewhite** [comment]
Jason and Luke's search solution will not be ready for this release, so we need to figure out how to support the current search solution in the `surge` branch and `rhdp2` theme

**Comment by jordanpagewhite** [comment]
@KamiQuasi is working to get the `rhdp-search-app` component transpiled into JS that we can consume in Drupal. Thanks!

**Comment by jordanpagewhite** [comment]
Marking blocked by https://github.com/redhat-developer/rhd-frontend/issues/296

**Comment by jordanpagewhite** [comment]
The 'Drupal' part of this issue is already done, as far as it is currently defined (based upon the current implementation), in that the web component is already rendered to the page via a WYSIWYG field. https://github.com/redhat-developer/rhd-frontend/issues/296 will track this issue to completion, so this ticket in the Drupal repo can be closed.
